.github/workflows/scheduled-istioCrypto-benchmark.yml

```diff
@@ -0,0 +1,195 @@
+              kubectl config view --minify --flatten > ~/minified_config
+              mv ~/minified_config ~/.kube/config
+              echo 'mesheryctl is not installed. Installing mesheryctl client... Standby... (Starting Meshery as well...)' >&2
+              # curl -L https://meshery.io/install | ADAPTERS=istio PLATFORM=kubernetes bash -
+
+              output=$(curl -L https://meshery.io/install | PLATFORM=kubernetes bash - 2>&1)
+              echo $output
+              url=$(echo "$output" | grep -oP 'http://\S+' | tail -1)
+              echo "Generated URL: $url"
+
+              #forces meshery to register components and checks 
+              curl --location "${url}/api/system/sync" \
+                  --header 'meshery-token:${{ secrets.MESHERY_TOKEN }}' \
+                  --header 'Cookie: meshery-provider=Meshery; meshery.layer5.io_ref=/;token=${{ secrets.MESHERY_TOKEN }}'
+
+              curl --location "${url}/api/system/sync" \
+                  --header 'meshery-token:${{ secrets.MESHERY_TOKEN }}' \
+                  --header 'Cookie: meshery-provider=Meshery; meshery.layer5.io_ref=/;token=${{ secrets.MESHERY_TOKEN }}'
+
+              check_endpoint() {
+                  response=$(curl -s "${url}/api/meshmodels/models/kubernetes/components/IstioOperator" \
+                       -H 'Cookie: meshery-provider=Meshery;token=${{ secrets.MESHERY_TOKEN }}'
+                      )
+                  total_count=$(echo "$response" | grep -oP '(?<="total_count":)\d+')
+                  
+                  if [[ $total_count -ne 0 ]]; then
+                      echo "Components Registered"
+                      exit 0
+                  fi
+              }
+
+              # Set the duration for script execution (in seconds)
+              duration=$((5 * 60))
+
+              # Calculate the end time
+              end_time=$((SECONDS + duration))
+
+              # Loop until the end time is reached
+              while [[ $SECONDS -lt $end_time ]]; do
+                  check_endpoint
+                  echo "Checked, Components not registered"
+                  sleep 60
+              done
+
+              echo "Waited for 5 minutes components not registered"
+
+      - name: Deploy Istio Crypto design and deploy httpbin application
+        run: |
+          chmod +x .github/workflows/scripts/istioCrypto.sh
+          .github/workflows/scripts/istioCrypto.sh
+        shell: bash
+
+      - name: Get Date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d-%H.%M.%S')"
+        
+      - name: Run Benchmark Tests
+        uses: layer5io/meshery-smp-action@master
+        with:
+          provider_token: ${{ secrets.MESHERY_TOKEN }}
+          platform: docker
+          profile_filename: load-test.yaml
+          endpoint_url: ${{env.ENDPOINT_URL}}
+          service_mesh: ${{env.SERVICE_MESH}}
+          load_generator: fortio
+          profile_name: 'istioCrypto-load-test.yaml'
```

Comment:
We need to add the certificate generated by `generate_csr.sh` script in the profile, we can add another item `cert_path` to use the certificate. But it depends on this PR:  https://github.com/meshery/meshery/pull/8148.